ext/filter/filter.stub.php

```diff
@@ -11,6 +11,10 @@ function filter_var(mixed $value, int $filter = FILTER_DEFAULT, array|int $optio
 /** @refcount 1 */
 function filter_input_array(int $type, array|int $options = FILTER_DEFAULT, bool $add_empty = true): array|false|null {}
 
+/**
+ * @return array<int|string, mixed|ref>|false|null
```

Comment:
It shouldn't be necessary to specify this type explicitly, `array<int|string, mixed|ref>` is the same as just `array`.